src/Tests/Creation/When_creating_queues.cs

```diff
@@ -453,5 +456,49 @@ await namespaceManager.CreateQueue(new QueueDescription("existingqueue")
             //cleanup
             await namespaceManager.DeleteQueue("existingqueue");
         }
+
+        [Test]
+        public async Task Should_not_update_properties_of_an_existing_shared_queue()
+        {
+            var queuePath = "errorQ";
+
+            var namespaceManager = new NamespaceManagerAdapter(NamespaceManager.CreateFromConnectionString(AzureServiceBusConnectionString.Value));
+            if (!await namespaceManager.QueueExists(queuePath).ConfigureAwait(false))
+            {
+                await namespaceManager.CreateQueue(new QueueDescription(queuePath)).ConfigureAwait(false);
+            }
+
+            var settings = new DefaultConfigurationValues().Apply(new SettingsHolder());
+            settings.Set("errorQueue", queuePath);
+            settings.Set("TypesToScan", new List<Type> {typeof(ConfigureAuditQueue)});
```

Comment:
@andreasohlund I worked around the issue. Saying that, `TryGet` should never fail. A different issue :)